website/docs/docs/collaborate/git/setup-azure.md

```diff
@@ -319,7 +324,8 @@ To connect the service user:
 
 Once connected, dbt Cloud displays the email address of the service user so you know which user's permissions are enabling headless actions in deployment environments. To change which account is connected, disconnect the profile in dbt Cloud, sign into the alternative Azure DevOps service account, and re-link the account in dbt Cloud.
 
-:::info Service user authentication expiration
-dbt Cloud will refresh the authentication for the service user on each run triggered by the scheduler, API, or CI. If your account does not have any active runs for over 90 days, an admin will need to manually refresh the authentication of the service user by disconnecting and reconnecting the service user's profile via the OAuth flow described above in order to resume headless interactions like project set up, deployment runs, and CI.
-
+:::info Personal Access Tokens (PATs)
+Full-scoped PATs are created within dbt Cloud for short access by the service user to hit APIs around dbt Cloud project setup. They are valid for only 5 minutes and are revoked after a call to the API has been made.
```

Comment:
```suggestion
Full-scoped PATs are created within dbt Cloud for short access by the service user to hit APIs around dbt Cloud project setup. They are valid for only 5 minutes and are revoked after a call to the API has been made.
:::
```

Reply:
Hey Mirna, this is more of an FYI that in the background these PAT tokens will be created automatically, not that a person can if they would like to.

Reply:
ah yes of course, i totally misread that! ok I've reworded it - does that look right?